doc/source/serve/package-ref.md

```diff
@@ -45,3 +45,63 @@
 ```{eval-rst}
 .. autofunction:: ray.serve.batch(max_batch_size=10, batch_wait_timeout_s=0.0)
 ```
+
+## Serve REST API
+
+### Config
+
+#### Schema
+
+```{eval-rst}
+.. autopydantic_model:: ray.serve.schema.ServeApplicationSchema
+
+```
+
+#### REST API
+
+```{eval-rst}
+.. http:GET:: "/api/serve/deployments/"
+    :noindex:
+
+        Gets latest config that Serve has received. This config represents the current goal state for the Serve application. Starts a Serve application on the Ray cluster if it's not already running. See Config Schema for the output schema.
```

Comment:
link to "config schema" and don't capitalize. also goes for other instances

Reply:
Good catch– I made the updates.